src/fuchsia/mod.rs

```diff
@@ -974,6 +974,11 @@ s_no_extra_traits! {
         pub sigev_notify_attributes: *mut pthread_attr_t,
         pub __pad: [::c_char; 56 - 3 * 8 /* 8 == sizeof(long) */],
     }
+
+    pub struct in6_pktinfo {
+        pub ipi6_addr: ::in6_addr,
+        pub ipi6_ifindex: ::c_int,
```

Comment:
Isn't it `c_uint`?

Reply:
you are [correct](https://cs.opensource.google/fuchsia/fuchsia/+/main:zircon/third_party/ulib/musl/include/netinet/in.h;l=308;drc=7c5e521391fddb98fd8f6970da7c410899ddf5cf).